queries/roc/textobjects.scm

```diff
@@ -0,0 +1,35 @@
+(anon_fun_expr
+  (expr_body) @function.inner
+) @function.outer
+
+(argument_patterns
+  ((_) @parameter.inner . ","? @parameter.outer) @parameter.outer
+)
+
+(function_type
+  ((_) @parameter.inner . ","? @parameter.outer) @parameter.outer(#not-eq? @parameter.inner "->")
```

Comment:
Same as above I believe. Also we don't enforce style (yet?) on this repo but would you mind putting the predicate on a new line, just some easier reading?

Reply:
We should add the query linter from `nvim-treesitter`; I don't think we should support a different style here. (You can just download/copy the file from that repo; it's a standalone script.)

Reply:
I added the query linter and ran it. I am still not sure what the issue is with this query though. If someone can explain to me in more detail I will make whatever change necessary.

https://github.com/nvim-treesitter/nvim-treesitter-textobjects/pull/593#discussion_r1554554398

Reply:
Oh, thank you for doing that! Sorry I should have mentioned this was done by me a few days ago; but I think your issue is that the linter expects the output to match the `main` branch formatter: I am guessing the one you added is from `master`?  (I also did this at first!) You can just remove that file from this PR and then run your original queries through the `main` branch's formatter and it should stop complaining :)